components/layout/flow_list.rs

```diff
@@ -93,6 +99,21 @@ impl FlowList {
     }
 }
 
+impl Index<usize> for FlowList {
+    /// FIXME(pcwalton): O(n)!
```

Comment:
:(.

This is only used for flexbox for now right? I wonder if we could rename it to `nth_slow` and `nth_slow_mut` so it's more obvious and so it's not misused without knowledge.